ecs/backend.go

```diff
@@ -60,11 +76,34 @@ func service(ctx context.Context) (backend.Service, error) {
 }
 
 func getEcsAPIService(ecsCtx store.EcsContext) (*ecsAPIService, error) {
+	var region string
+	var profile string
+
+	if ecsCtx.CredentialsFromEnv {
+		creds := getEnvVars()
+		if !creds.haveRequiredCredentials() {
+			return nil, fmt.Errorf(`context requires credentials to be passed as environment variable`)
```

Comment:
nit: we don't need backquotes here
```suggestion
			return nil, fmt.Errorf("context requires credentials to be passed as environment variable")
```